lib/version_utils.pm

```diff
@@ -239,6 +240,10 @@ sub is_rt {
     return check_var('SLE_PRODUCT', 'rt');
 }
 
+sub is_released {
+    return get_var('FLAVOR') =~ /Incidents/ || get_var('FLAVOR') =~ /Updates/;
```

Comment:
That seems to be the same as https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/e70ba1caa4403c93f230ed94279c0971d7f8f933/lib/main_common.pm#L286 now. I recommend to clean this up a bit.